internal/cli/atlas/backup/snapshots/__snapshots__/list_test.snap

```diff
@@ -0,0 +1,13 @@
+
+[TestList_Run - 1]
```

Comment:
This is snapshot generated using faker (As we see it is not giving us real data)